### DISCUSSION TITLE
Jellyfin演员元数据文件夹没有nfo文件

### DISCUSSION THREAD

**Discussion by haoylee** [sqzw-x/mdcx#162] [Q&A]
以前emby是有的，演员nfo文件和照片都在metadata的people文件夹内，现在更新4.8.3.0之后也没有了（他们说要勾选使用nfo文件选项，我勾了还是没用）。这样导致一个问题，以前我在emby页面内修改演员信息，修改都可以同步到nfo文件内，方便重装时备份或迁移，现在却只有头像了，然后可以单向地自己上传nfo文件到这个文件夹，可以更新演员信息，但反过来在网页上操作修改演员信息则完全无法作用到nfo文件上。所以我猜测是有一个隐藏的演员元数据文件，较新的修改只会更新到这个内置元数据里。

不知道对于本来就没有演员nfo文件的jellyfin来说，是不是原理也是这样？那么刮削的演员信息到底是存在哪里的？有办法在服务器系统中找出来么？（我的服务器是mac系统）

我本来就是因为jellyfin备份不出来演员的nfo才转用emby的，结果现在emby也不行了，虽然貌似设置里还有这个选项开启，但是我开启了没有作用啊，抓狂了。。

这是emby的设置，有选项但是现在不起作用，论坛里也有其他人报告了这个bug
![未命名](https://github.com/sqzw-x/mdcx/assets/37681300/d15d26d5-97a8-4104-9302-e2c77e0324df)

jellyfin的设置里没有选项，也一直没有演员nfo文件
![未命名2](https://github.com/sqzw-x/mdcx/assets/37681300/0b1b820e-129a-44a3-bea0-1181f69e05c5)

![未命名](https://github.com/sqzw-x/mdcx/assets/37681300/24162d78-ddb1-4992-8bfe-407a54f19c1e)

**Comment by sqzw-x** [comment]
emby 的所有元数据都是保存在数据库里的，而且是同一个表，这个表里同时包含了媒体、标签、合集、演员的信息，非常混乱
具体而言，Windows 上是在 `%AppData%\Emby-Server\programdata\data\library.db` 内
people 内我看了一下，原来是有 nfo 的，不过最近添加的都没有，应该是某个版本后改变了存储方式，现在这个目录应该只保存图片

**Reply by sqzw-x** [reply]
另外通过 emby 的 api 是可以获取演员信息的，不过需要自行查阅 api 文档了

**Comment by haoylee** [comment]
这个问题我问过emby论坛里的官方人员了，按理说emby开了将演员数据保存到nfo的话应该还是会在people文件夹里有nfo的，跟之前一样，但是可能这是新更新的机制，所以没成功，下一个版本有可能修复好
